openapi/clusters_mgmt/v1/openapi.json

```diff
@@ -16433,17 +16437,73 @@
         }
       },
       "ExternalAuthConfig": {
-        "description": "ExternalAuthConfig configuration",
+        "description": "Represents an external authentication configuration",
         "properties": {
+          "kind": {
+            "description": "Indicates the type of this object. Will be 'ExternalAuthConfig' if this is a complete object or 'ExternalAuthConfigLink' if it is just a link.",
+            "type": "string"
+          },
+          "id": {
+            "description": "Unique identifier of the object.",
+            "type": "string"
+          },
+          "href": {
+            "description": "Self link.",
+            "type": "string"
+          },
           "enabled": {
-            "description": "Boolean flag indicating if the cluster should use an external authentication configuration.\n\nBy default this is false.\n\nTo enable it the cluster needs to be ROSA HCP cluster and the organization of the user needs\nto have the `external-authentication` feature toggle enabled.",
+            "description": "Boolean flag indicating if the cluster should use an external authentication configuration.\n\nBy default this is false.\n\nTo enable it the cluster needs to be ROSA HCP cluster and the organization of the user needs\nto have the `external-authentication` feature toggle enabled.\n\nFor ARO HCP clusters, please use the \"State\" property to enable/disable this feature instead.",
```

Comment:
Not sure if this is possible but can the description be different for commercial vs ARO HCP?